sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -356,10 +356,12 @@ def get_queue_receiver(
          will be immediately removed from the queue, and cannot be subsequently rejected or re-received if
          the client fails to process the message. The default receive_mode is PEEK_LOCK.
         :paramtype receive_mode: Union[~azure.servicebus.ServiceBusReceiveMode, str]
-        :keyword Optional[float] max_wait_time: The timeout in seconds between received messages after which the
-         receiver will automatically stop receiving. The default value is None, meaning no timeout. If connection
-         errors are occurring due to write timing out, the connection timeout value may need to be adjusted. See
-         the `socket_timeout` optional parameter for more details.
+        :keyword Optional[float] max_wait_time:  The timeout in seconds to wait for the first and subsequent
+         messages to arrive after which the receiver will automatically stop receiving. If no messages arrive,
+         and no timeout is specified, this call will not return until the connection is closed.
+         The default value is None, meaning no timeout. If connection errors are occurring due to write timing out,
+         the connection timeout value may need to be adjusted. See the `socket_timeout`
```

Comment:
I'm not sure if this comment is necessarily for this PR, but do we think having "connection timeout value may need to be adjusted. See `socket_timeout`" might be confusing. Instead of "socket timeout value may need to be adjusted". It makes it seem like connection timeout could be yet another timeout, not socket timeout. Thoughts?

Reply:
I agree it does sound like it is saying to change a separate timeout value

Reply:
agree it can be a simple change like socket connection timeout value may need to be adjusted